src/main/java/com/moodybluez/enterprise/dao/EntrySQLDAO.java

```diff
@@ -14,26 +17,28 @@
     @Autowired
     private EntryRepository entryRepository;
 
+    Logger LOG = LoggerFactory.getLogger(this.getClass());
+
```

Comment:
Defining Logger to be used later for better logging of issues in program. It is generally a Best practice to use Logging rather than just System.out.println or even just leaving it blank.